lib/examples/compare-chromedp/click/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/ysmood/rod"
+	"log"
+	"time"
+)
+
+// This example demonstrates how to use a selector to click on an element.
+func main() {
+	browser := rod.New().Timeout(15 * time.Second).Trace(true).Connect()
```

Comment:
`Trace(true)` maybe not necessary.

Reply:
I used Trace here because the chromedp example does. I am replicating the output :)